test/composer.js

```diff
@@ -356,7 +356,7 @@ test.cb('Composer.lazy should support middlewares', (t) => {
 })
 
 test.cb('Composer.dispatch should work with handlers array', (t) => {
-const bot = createBot()
+  const bot = createBot()
```

Comment:
I assume you had no _intent to indent_ (pun indented)

```suggestion
const bot = createBot()
```